docs/GettingStarted.md

```diff
@@ -1935,6 +1961,10 @@ end
 
 The Sneakers integration is a server-side middleware which will trace job executions.
 
+<div class="alert alert-warning">
+Kicks is a continuation of Sneakers and both cannot be active at the same time, as they share their Ruby class namespace. Configurations to the Sneakers integration will be safely merged with any Kicks configuration.
```

Comment:
```suggestion
Kicks is a continuation of Sneakers. Because Kicks and Sneakers share a Ruby class namespace, they cannot both be active at the same time. Configurations to the Sneakers integration is safely merged with any Kicks configuration.
```